seatunnel-connectors-v2/connector-jdbc/pom.xml

```diff
@@ -122,6 +123,12 @@
                 <version>${saphana.version}</version>
                 <scope>provided</scope>
             </dependency>
+
+            <dependency>
+                <groupId>net.snowflake</groupId>
+                <artifactId>snowflake-jdbc</artifactId>
+                <version>${snowflake.version}</version>
```

Comment:
Please make scope is `provided`.
And please add this dependency in `seatunnel-dist/pom.xml` like other jdbc drivers.

Reply:
just added @hailin0